### DISCUSSION TITLE
Your templates

### DISCUSSION THREAD

**Discussion by TonyWinyard** [BryanJenksCommunity/FAQ#147] [Ideas & Suggestions]
Thank you very much for the Obsidian templates.
I've only just started using Obsidian, and so still struggling slightly. Do you have any type of document guideline for the templates?
On looking through the various templates, there are many I have no idea what some of the content in the template is for, or how it might be used.

**Comment by tallguyjenks** [comment]
i dont have a guide up right now, I've shown many of the templates in various videos but its not quite organized yet, I'll likely make a guide in the future, I'm just incredibly busy right now 😅  sorry!

**Comment by KnikmanAV** [comment]
a guide would be very helpful :)
